src/en/kagane/src/eu/kanade/tachiyomi/extension/en/kagane/Dto.kt

```diff
@@ -31,84 +31,68 @@ class SearchDto(
 
 @Serializable
 class DetailsDto(
-    val data: Data,
+    val source: String,
+    val authors: List<String>,
+    val status: String,
+    val summary: String?,
+    val genres: List<String>,
+    @SerialName("alternate_titles")
+    val alternateTitles: List<AlternateTitles>,
 ) {
     @Serializable
-    class Data(
-        val metadata: Metadata,
-        val source: String,
-    ) {
-        @Serializable
-        class Metadata(
-            val genres: List<String>,
-            val status: String,
-            val summary: String,
-            val alternateTitles: List<Title>,
-        ) {
-            @Serializable
-            class Title(
-                val title: String,
-            )
-        }
-
-        fun toSManga(): SManga = SManga.create().apply {
-            val summary = StringBuilder()
-            summary.append(metadata.summary)
-                .append("\n\n")
-                .append("Source: ")
-                .append(source)
+    class AlternateTitles(
+        val title: String,
+    )
 
-            if (metadata.alternateTitles.isNotEmpty()) {
-                summary.append("\n\nAssociated Name(s):")
-                metadata.alternateTitles.forEach { summary.append("\n").append("• ${it.title}") }
-            }
+    fun toSManga(): SManga = SManga.create().apply {
+        val desc = StringBuilder()
+        if (!summary.isNullOrBlank()) desc.append(summary + "\n\n")
+        desc.append("Source: ").append(source + "\n\n")
 
-            description = summary.toString()
-            genre = metadata.genres.joinToString()
-            status = metadata.status.toStatus()
+        if (alternateTitles.isNotEmpty()) {
+            desc.append("Associated Name(s):\n\n")
+            alternateTitles.forEach { desc.append("• ${it.title}\n") }
         }
 
-        private fun String.toStatus(): Int {
-            return when (this) {
-                "ONGOING" -> SManga.ONGOING
-                else -> SManga.COMPLETED
-            }
+        author = authors.joinToString { it }
```

Comment:
Couldn't you just do this?
```suggestion
        author = authors.joinToString()
```

\* not tested

Reply:
Yep, maybe I was testing something and I left it there, changed.